### PR TITLE
A script for sending reciepts for recurring monthly donations on stri…

### PR DESCRIPTION
**Pull request by cadecairos**
…pe (that occurred before a specified data)

#998 

`BEFORE_DATE="2015-11-28T03:50:58Z" node scripts/send-subscription-receipts`

@jbuck R?

### CODE REVIEW

File: scripts/send-subscription-receipts.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+require('habitat').load();
+
+var stripe = require('stripe')(process.env.STRIPE_SECRET_KEY);
+var hatchet = require('hatchet');
+var async = require('async');
+var moment = require('moment');
+
+var stripe_charge_list_opts = {
+  created: {
+    lte: moment.unix(new Date(process.env.BEFORE_DATE)).valueOf()
```
Comment (jbuck) [inline scripts/send-subscription-receipts.js]: This needs to be converted to a regular Unix epoch count in seconds, not the Javascript variant that counts in ms. You could remove the moment dependency and change it to:

`new Date(process.env.BEFORE_DATE).valueOf() / 1000`

**Comment by jbuck** [comment]
@cadecairos this works, but walking all 51,892 charges, 100 charges at a time is going to be slow. I think we should modify mofo-transaction-storage to store where a transaction is recurring or not so we can do this more quickly...

**Comment by jbuck** [comment]
@cadecairos `select id from stripe where recurring = TRUE and timestamp <= '2015-11-28T03:50:58Z';`

**Comment by jbuck** [comment]
@cadecairos so basically we can just add the list of ids as an array to this script, and run that through

**Comment by cadecairos** [comment]
Lovely!

**Comment by cadecairos** [comment]
oh wow, I was working on my master branch! derp

**Pull request closed without merging**